CHANGELOG.md

```diff
@@ -10,6 +10,8 @@ and this project's packages adheres to [Semantic Versioning](http://semver.org/s
 ### Fixed
 - Prevent breaking the client's kubeconfig if token renewal fails.
 
+## [0.6.0] - 2020-07-24
+Adds `--use-alike-instance-types` for node pools
```

Comment:
```suggestion
### Added
Adds `--use-alike-instance-types` for node pools

```

Reply:
Automation will move the unreleased change to the right place in the CI

Reply:
Made the changes, is this correct?